rwilliams659/rancid-tomatillos
# Rw favoriting filtering

#### What changes/additions are a part of this PR?
- Add new Favorites page that only displays favorited movie; this uses existing Movies/movie components instead of new component since it's the same content 
- Add Favorites page to Nav bar
 -If no movies are favorited, display message telling users to add favorites 
- Update the homepage so that once users are logged in, they can favorite or unfavorite a movie by clicking the heart icon
- Update the movie details view so that once users are logged in, they can favorite or unfavorite a movie by clicking the heart icon

- Based on changes made to existing components to incorporate favoriting feature, alter test files

#### Any background context you want to provide?
There are still a few failing tests in MovieDetails but everything else should be working now. 

#### Where should the reviewer start?
App component to view the new Route for the Favorites page, then Movies & Movie to view the changes made there to enable the Favoriting icons to render/favoriting to work 

#### How should this be tested (test suite/manually)?
Run the test suites! Also take a look at the app manually - once you log in, you should see the favoriting icons

#### What are the relevant tickets?
#47 , #48 , #49 , #53 

#### Screenshots (if helpful):
![Screen Shot 2020-08-31 at 3 38 52 PM](https://user-images.githubusercontent.com/62770765/91771441-400b5280-eba0-11ea-87a8-2a6737e1f644.png)
![Screen Shot 2020-08-31 at 3 39 11 PM](https://user-images.githubusercontent.com/62770765/91771451-44377000-eba0-11ea-9636-1acde98f109d.png)
![Screen Shot 2020-08-31 at 3 39 45 PM](https://user-images.githubusercontent.com/62770765/91771470-48638d80-eba0-11ea-8bf5-e17a9b76b6b7.png)
